File: Lecture22: Strings/replaceSpaces.cpp
#include<string>
#include<iostream>
using namespace std;
string replaceSpaces(string &str){
	string temp = "";
    
    for(int i=0; i<str.length(); i++) {
        if(str[i] == ' '){
            temp.push_back('@');
            temp.push_back('4');
            temp.push_back('0');
        }
        else
        {
            temp.push_back(str[i]);
        }
    }
    return temp;
}

File: Lecture22: Strings/replaceSpaces.java
public static String replaceSpaces(String str){
	String temp = "";
    
    for(int i=0; i<str.length(); i++) {
        if(str.charAt(i) == ' '){
            temp += '@';
            temp += '4';
            temp += '0';
        }
        else
        {
            temp += str.charAt(i);
        }
    }
    return temp;
}
